packages/core/src/lavadome.mjs

```diff
@@ -41,21 +46,25 @@ export function LavaDome(host, opts) {
     const child = hardened();
     appendChild(shadow, child);
 
-    function text(text) {
-        if (typeof text !== 'string') {
+    let secret = '';
+
+    function text(input) {
+        if (typeof input !== 'string') {
             throw new Error(
-                `LavaDomeCore: first argument must be a string, instead got ${stringify(text)}`);
+                `LavaDomeCore: first argument must be a string, instead got ${stringify(input)}`);
```

Comment:
```suggestion
                `LavaDomeCore: first argument must be a string, instead got ${typeof input}`);
```
Type should be enough, no need to template secret in error message

Reply:
5265dd425bf11a1f9ef7dc5511366bd87cd47412

Reply:
also 38ef179